src/extension.ts

```diff
@@ -281,16 +279,10 @@ export class Extension {
       }
 
       for (const project of model.projects.values()) {
-        await this._createRunProfile(project, usedProfiles);
+        await this._createRunProfile(project, isFirstProject);
```

Comment:
Now every time we rebuild the model user settings will be reset. We should restore them.

Reply:
They also used to persist across the sessions, I wonder what happened to that use case.